engine/execution/computation/manager_test.go

```diff
@@ -292,18 +288,15 @@ func TestExecuteScript_ShortScriptsAreNotLogged(t *testing.T) {
 	buffer := &bytes.Buffer{}
 	log := zerolog.New(buffer)
 
-	view := delta.NewView(func(_, _, _ string) (flow.RegisterValue, error) {
-		return nil, nil
-	})
 	header := unittest.BlockHeaderFixture()
 
 	eds := new(state_synchronization.ExecutionDataService)
 	edCache := new(state_synchronization.ExecutionDataCIDCache)
 
-	manager, err := New(log, metrics.NewNoopCollector(), nil, nil, nil, vm, ctx, DefaultProgramsCacheSize, committer.NewNoopViewCommitter(), 1*time.Second, nil, eds, edCache)
+	manager, err := New(log, metrics.NewNoopCollector(), nil, nil, nil, vm, ctx, DefaultProgramsCacheSize, committer.NewNoopViewCommitter(), 1*time.Second, DefaultScriptExecutionTimeLimit, nil, eds, edCache)
```

Comment:
Maybe chop down the argument list here, or even consider defining a struct which bundles all the parameters of the function

Reply:
did chop down, we are going to revisit all these tests soon, so we could consider using a param object as we refactor.